docs/source/guides/localisation.rst

```diff
@@ -1,14 +1,101 @@
+a ``"Token"`` key indexing all translations.
+
+.. warning ::
+    Translations files must use ``"UTF-16 LE"`` encoding.
+
+You'll have to create one file per supported language, and all your files must be named in a similar fashion.
+
+For example, Northstar translation files are named ``"northstar_client_localisation_english.txt"``, ``"northstar_client_localisation_french.txt"``, 
+``"northstar_client_localisation_german.txt"`` etc.
+
+You can import them from your ``mod.json`` manifesto this way:
+
+.. code-block:: json
+
+    {
+        "Localisation": [
+            "resource/northstar_client_localisation_%language%.txt"
+        ]
+    }
 
-Files to translate
-------------------
+.. note::
+    The ``"%language%"`` syntax allows VM to load up translations matching game language (e.g. an English client will automatically use 
+    ``"northstar_client_localisation_english.txt"`` file)
+
+Use translations in your code
+-----------------------------
+
+To translate UI elements like menus, you have to insert strings containing your translation keys, preceded by a ``#``.
+
+For example, to translate the "Launch Northstar" button on main menu, instead of calling:
+
+.. code-block:: javascript
+
+    AddComboButton( comboStruct, headerIndex, buttonIndex++, "Launch Northstar" )
+
+We'll use:
+
+.. code-block:: javascript
+
+    AddComboButton( comboStruct, headerIndex, buttonIndex++, "#MENU_LAUNCH_NORTHSTAR" )
+
+You can also use the ``Localize`` method client-side:
+
+.. code-block:: javascript
+
+    Localize( "#MENU_LAUNCH_NORTHSTAR" )
+
+Northstar translations
+----------------------
+
+Northstar adds new strings to the game which can be localised to match the language you are using on your Titanfall 2 installation.
 
-There are the two files that need to be translated, use the English version as a base. After you finish make sure that the files are encoded in `UTF-16 LE` before opening a Pull Request.
+They're all located in ``"Northstar.Client"`` mod: `Northstar localisation files on GitHub <https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Client/mod/resource>`_
 
-1. `Northstar.Client/mod/resource/northstar_client_localisation_english.txt <https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Client/mod/resource/northstar_client_localisation_english.txt>`_
-2. `Northstar.Custom/mod/resource/northstar_custom_english.txt <https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Custom/mod/resource/northstar_custom_english.txt>`_
+.. note ::
+    Don't hesitate to submit a PR if you notice a typo!
 
-To test your modifications go to `Origin (My games library) -> Titanfall 2 (right click) -> Game Properties -> Advanced Launch Options` and select the language you modified from the dropdown.
\ No newline at end of file
+    To test your modifications go to `Origin (My games library) -> Titanfall 2 (right click) -> Game Properties -> Advanced Launch Options` and select the language you modified from the dropdown.
```

Comment:
Maybe add a section on how to do this in steam, if that even possible.

Reply:
I'll check by tonight.

Reply:
Fixed with 9e83299b118d106e925cb0ba194a890e275a98b2.